docs/pages/pro-features/generic-resource-patches.mdx

```diff
@@ -0,0 +1,114 @@
+export: []
+import: []
+```
+
+The `apiVersion` field expects a "group/version" definition of both the resource group and API version.
+The `kind` field specifies which Kubernetes resources to patch.
+The `verbs` field defines the API calls to patch before application to the Kubernetes API.
+The `patches` reuse the same [syntax as the patches](../architecture/synced-resources#patch-syntax) utilized in [generic sync](../architecture/synced-resources#generic-sync).
+
+```yaml
+version: v1beta1
+hooks:
+  hostToVirtual:
+    - apiVersion: v1
+      kind: Pod
+
+      patches:
+        - op: add
+          path: metadata.annotations
+          value:
+            import-annotation: from-host-to-virtual
+
+  virtualToHost: []
+
+# Generic Sync Configs
+export: []
+import: []
+```
+
+## Example - Node Info Remover
+
+The example below removes all `.status.images` and all `.metadata.annotations` except the `node.alpha.kubernetes.io/ttl` annotation, all labels except some `kubernetes.io` labels.
+
+```yaml
+version: v1beta1
+hooks:
+  hostToVirtual:
+    - apiVersion: v1
+      kind: Node
+
+      verbs: ["update", "patch", "create", "get", "list"]
+
+      patches:
+        - op: remove
+          path: status.images
+        - op: replace
+          path: metadata.annotations
+          value: {}
+        - op: replace
+          path: metadata.labels
+          value: {}
+        - op: copyFromObject
+          fromPath: metadata.annotations["node.alpha.kubernetes.io/ttl"]
+          path: metadata.annotations["node.alpha.kubernetes.io/ttl"]
+
+        # Add arch labels
+        - op: copyFromObject
+          fromPath: metadata.labels["kubernetes.io/arch"]
+          path: metadata.labels["kubernetes.io/arch"]
+        - op: copyFromObject
+          fromPath: metadata.labels["kubernetes.io/hostname"]
+          path: metadata.labels["kubernetes.io/hostname"]
+        - op: copyFromObject
+          fromPath: metadata.labels["kubernetes.io/os"]
+          path: metadata.labels["kubernetes.io/os"]
+        - op: copyFromObject
+          fromPath: metadata.labels["beta.kubernetes.io/arch"]
+          path: metadata.labels["beta.kubernetes.io/arch"]
+        - op: copyFromObject
+          fromPath: metadata.labels["beta.kubernetes.io/hostname"]
+          path: metadata.labels["beta.kubernetes.io/hostname"]
+        - op: copyFromObject
+          fromPath: metadata.labels["beta.kubernetes.io/os"]
+          path: metadata.labels["beta.kubernetes.io/os"]
+
+  virtualToHost: []
+```
+
+## Usage in Pro Helm Chart
+
+When installing vcluster pro, make sure to set your generic sync configs in the `sync.generic.config` field as string.
+
+For example by either inlining it during helm install/upgrade
+
+```bash
+helm upgrade [RELEASE_NAME] loft-sh/vcluster-pro -n [RELEASE_NAMESPACE] --create-namespace --install --set sync.generic.config="..."
+```
+
+Or by specifying it in a dedicated values.yaml
+
+```yaml
+sync:
+  generic:
+   config: |–
+     ---
```

Comment:
Probably a good idea to add the config starting point instead of empty `---`. Gives a better idea to the reader where to fit in the stringified config

```suggestion
   config: |–
     version: v1beta1
     hooks:
       hostToVirtual:
       ...
```